packages/platform/src/types/Mutation.ts

```diff
@@ -25,11 +25,13 @@ interface GenerateBaseMutationsArgs {
   reducers?: any
   schemas?: any
   inputTypes?: any
+  // TODO(JJ): is returning void right here? It should return a UserRole, no?
   roleAssigner?: (ix: number) => void
 }
 
 export function generateBaseMutations<PeriodFacts, PeriodSegmentFacts>({
   reducers = defaultReducers,
+  // TODO(JJ): What Is Unset? Why string?
   roleAssigner = () => 'UNSET',
```

Comment:
the enum is basically a string. but we should just not set roleAssigner in the base mutations and make our code so it works also without user roles (or make PLAYER the default)

Reply:
-> to be continued and discussed

Reply:
As discussed, roleAssigner is undefined if not provided.